components/director/pkg/graphql/runtime_validation_test.go

```diff
@@ -26,9 +26,9 @@ func TestRuntimeInput_Validate_Name(t *testing.T) {
 			ExpectedValid: false,
 		},
 		{
-			Name:          "Invalid - Invalid Name",
+			Name:          "Expected valid with digit",
 			Value:         inputvalidationtest.InvalidName,
-			ExpectedValid: false,
+			ExpectedValid: true,
```

Comment:
Change the `inputvalidationtest.InvalidName` to be invalid instead of removing the invalid test.